cores/nRF5/sysview/SEGGER/SEGGER_RTT.c

```diff
@@ -240,7 +235,7 @@ Additional information:
 **********************************************************************
 */
 
-static unsigned char _aTerminalId[16] = { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F' };
+static unsigned char _aTerminalId[SEGGER_RTT_NUMBER_OF_TERMINALS] = { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F' };

```

Comment:
Please do not modify the original RTT file from SEGGER. We should leave it as it is to make it effort-free maintenance